### PR TITLE
Update strings.xml

### PR DESCRIPTION
**Pull request by ThomasLeigh**

### CODE REVIEW

File: FlymFork/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -392,7 +392,7 @@
     <string name="text">Text</string>
     <string name="background1">Background</string>
     <string name="textColor">Text color</string>
-    <string name="textColorRead">Text color read</string>
+    <string name="textColorRead">Color of articles which have been read</string>
```
Comment (yanus171) [inline FlymFork/src/main/res/values/strings.xml]: Too long for prerefence title. It will be cut. Try to shorten it

Comment (ThomasLeigh) [reply FlymFork/src/main/res/values/strings.xml]: I think Your initial idea is better, so let;'s leave _"Read"_ and _"Read articles"_.

**Comment by SIRSteiner** [comment]
Look at https://github.com/yanus171/Handy-News-Reader/pull/221
Therefore you should close this PR

**Comment by yanus171** [comment]
@ThomasLeigh, will you pull request Read text color?

**Pull request closed without merging**